src/app/pages/send/send.component.tsx

```diff
@@ -227,6 +233,32 @@ export class SendPage extends Component<IProps, IState> {
         );
     }
 
+    public getAddressFromName(name) {
```

Comment:
This would be more suitable in a `service` or `helper` function. Shouldn't be part of the view layer. @krisboit what do you think?